app/brief_utils.py

```diff
@@ -36,3 +37,17 @@ def get_supplier_service_eligible_for_brief(supplier, brief):
     services = services.filter(Service.supplier_id == supplier.supplier_id)
 
     return services.first()
+
+
+def index_brief(brief):
+    if (
+        brief.framework.framework == 'digital-outcomes-and-specialists' and
```

Comment:
I'm not sure I see why we need this constraint. We only have DOS briefs anyway, and if there was another framework, then we'd need to do a whole heap of work to decide where those briefs lived _anyway_. They might well want to end up in the same index, I don't think we know?

Reply:
I was trying to stay in keeping with what was being done when indexing g-cloud services. I guess the check is there to prevent something like a g-cloud service being indexed as a brief by mistake. Happy to remove if you don't think it's necessary?

Reply:
So hopefully no-one will call this with a service, I assume that's fairly not-possible.

For services, perhaps this ought now to be controlled purely by `DM_FRAMEWORK_TO_ES_INDEX_MAPPING`?

Of course that's a bit of a contradiction with the other conversation, where I'm claiming a missing mapping should be an error - this implies we should just ignore it. Ho hum...

Reply:
Yeah, it's not a particularly helpful check. I'll get rid of it.